src/ui/pages/History/ActionDetailedView/components/FeeLine.tsx

```diff
@@ -41,24 +41,36 @@ export function FeeLine({
       justifyContent="space-between"
       style={{ gridTemplateColumns: 'auto 1fr' }}
     >
-      <UIText kind="small/regular">Fee</UIText>
+      <UIText kind="small/regular">Network Fee</UIText>
       <UIText kind="small/accent" style={{ justifySelf: 'end' }}>
-        <HStack gap={4}>
-          <span>{formatTokenValue(feeEth, '')}</span>
-          {nativeAsset.id ? (
-            <AssetLink
-              asset={{
-                asset_code: nativeAsset.id,
-                name: nativeAsset.name,
-                symbol: nativeAsset.symbol,
-              }}
-              address={address}
-            />
-          ) : (
-            nativeAsset.symbol?.toUpperCase()
-          )}
-          <span>({formatCurrencyValue(feeCurrency, 'en', 'usd')})</span>
-        </HStack>
+        {fee.quantity == '0' ? (
```

Comment:
is ` == '0'` a typo or is it something like `== null`?

Reply:
Intentional, but fortunately backend introduced a dedicated property for this, so I'll update